data/human/post-war reactions.txt

```diff
@@ -224,25 +218,24 @@ mission "War Reactions: Syndicate Orphan"
 				`	"I blew up my share of Pug ships."`
 					goto blew
 				`	"Actually, I led the battle."`
-			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device.' After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you. Can I shake your hand?" He puts out his hand, and you shake it.`
```

Comment:
Because the player isn't necessarily a "sir"

Reply:
Huh, after a quick search of current conventions, apparently making people constantly evaluate and assume gender is indeed actually being reinforced in various militaries again instead of phased out after all. Sigh.

Yea, that makes sense then.